tests/raw_to_standard/icone_test_expected_results.py

```diff
@@ -0,0 +1,595 @@
+test_generate_standard_messages_from_string_expected = [
```

Comment:
This is probably fine as a python file, but I'm curious why a python file and not just have a directory with expected JSON files then load the JSON as needed?